diff --git a/CRT.c b/CRT.c
index 85f2962..4f2b701 100644
--- a/CRT.c
+++ b/CRT.c
@@ -606,7 +606,92 @@ static int CRT_colorSchemes[LAST_COLORSCHEME][LAST_COLORELEMENT] = {
       [ZFS_RATIO] = ColorPair(Magenta, Black),
       [ZRAM] = ColorPair(Yellow, Black),
    },
-   [COLORSCHEME_BROKENGRAY] = { 0 } // dynamically generated.
+   [COLORSCHEME_BROKENGRAY] = { 0 }, // dynamically generated.
+   [COLORSCHEME_KEES] = {
+      [RESET_COLOR] = ColorPair(White, Black),
+      [DEFAULT_COLOR] = ColorPair(White, Black),
+      [FUNCTION_BAR] = ColorPair(Black, Magenta),
+      [FUNCTION_KEY] = ColorPair(White, Black),
+      [PANEL_HEADER_FOCUS] = ColorPair(Black, Magenta),
+      [PANEL_HEADER_UNFOCUS] = ColorPair(Black, Magenta),
+      [PANEL_SELECTION_FOCUS] = ColorPair(Black, Cyan),
+      [PANEL_SELECTION_FOLLOW] = ColorPair(Black, Yellow),
+      [PANEL_SELECTION_UNFOCUS] = ColorPair(Black, White),
+      [FAILED_SEARCH] = ColorPair(Red, Cyan),
+      [FAILED_READ] = A_BOLD | ColorPair(Red, Black),
+      [PAUSED] = A_BOLD | ColorPair(Yellow, Cyan),
+      [UPTIME] = A_BOLD | ColorPair(Magenta, Black),
+      [BATTERY] = A_BOLD | ColorPair(Magenta, Black),
+      [LARGE_NUMBER] = A_BOLD | ColorPair(Red, Black),
+      [METER_TEXT] = ColorPair(Magenta, Black),
+      [METER_VALUE] = A_BOLD | ColorPair(Magenta, Black),
+      [METER_VALUE_ERROR] = A_BOLD | ColorPair(Red, Black),
+      [METER_VALUE_IOREAD] = ColorPair(Magenta, Black),
+      [METER_VALUE_IOWRITE] = ColorPair(Blue, Black),
+      [METER_VALUE_NOTICE] = A_BOLD | ColorPair(White, Black),
+      [METER_VALUE_OK] = ColorPair(Magenta, Black),
+      [METER_VALUE_WARN] = A_BOLD | ColorPair(Yellow, Black),
+      [LED_COLOR] = ColorPair(Magenta, Black),
+      [TASKS_RUNNING] = A_BOLD | ColorPair(Magenta, Black),
+      [PROCESS] = A_NORMAL,
+      [PROCESS_SHADOW] = A_BOLD | ColorPairGrayBlack,
+      [PROCESS_TAG] = A_BOLD | ColorPair(Yellow, Black),
+      [PROCESS_MEGABYTES] = ColorPair(Cyan, Black),
+      [PROCESS_GIGABYTES] = ColorPair(Green, Black),
+      [PROCESS_BASENAME] = A_BOLD | ColorPair(Cyan, Black),
+      [PROCESS_TREE] = ColorPair(Cyan, Black),
+      [PROCESS_R_STATE] = ColorPair(Green, Black),
+      [PROCESS_D_STATE] = A_BOLD | ColorPair(Red, Black),
+      [PROCESS_HIGH_PRIORITY] = ColorPair(Red, Black),
+      [PROCESS_LOW_PRIORITY] = ColorPair(Green, Black),
+      [PROCESS_NEW] = ColorPair(Black, Green),
+      [PROCESS_TOMB] = ColorPair(Black, Red),
+      [PROCESS_THREAD] = ColorPair(Magenta, Black),
+      [PROCESS_THREAD_BASENAME] = A_BOLD | ColorPair(Green, Black),
+      [PROCESS_COMM] = ColorPair(Magenta, Black),
+      [PROCESS_THREAD_COMM] = ColorPair(Blue, Black),
+      [BAR_BORDER] = A_BOLD,
+      [BAR_SHADOW] = A_BOLD | ColorPairGrayBlack,
+      [SWAP] = ColorPair(Red, Black),
+      [GRAPH_1] = A_BOLD | ColorPair(Cyan, Black),
+      [GRAPH_2] = ColorPair(Cyan, Black),
+      [MEMORY_USED] = ColorPair(Magenta, Black),
+      [MEMORY_BUFFERS] = ColorPair(Blue, Black),
+      [MEMORY_BUFFERS_TEXT] = A_BOLD | ColorPair(Blue, Black),
+      [MEMORY_CACHE] = ColorPair(Yellow, Black),
+      [LOAD_AVERAGE_FIFTEEN] = ColorPair(Magenta, Black),
+      [LOAD_AVERAGE_FIVE] = A_BOLD | ColorPair(Magenta, Black),
+      [LOAD_AVERAGE_ONE] = A_BOLD | ColorPair(Magenta, Black),
+      [LOAD] = A_BOLD,
+      [HELP_BOLD] = A_BOLD | ColorPair(Cyan, Black),
+      [CLOCK] = A_BOLD,
+      [DATE] = A_BOLD,
+      [DATETIME] = A_BOLD,
+      [CHECK_BOX] = ColorPair(Cyan, Black),
+      [CHECK_MARK] = A_BOLD,
+      [CHECK_TEXT] = A_NORMAL,
+      [HOSTNAME] = A_BOLD,
+      [CPU_NICE] = ColorPair(Blue, Black),
+      [CPU_NICE_TEXT] = A_BOLD | ColorPair(Blue, Black),
+      [CPU_NORMAL] = ColorPair(Magenta, Black),
+      [CPU_SYSTEM] = ColorPair(Magenta, Black),
+      [CPU_IOWAIT] = A_BOLD | ColorPairGrayBlack,
+      [CPU_IRQ] = ColorPair(Yellow, Black),
+      [CPU_SOFTIRQ] = ColorPair(Magenta, Black),
+      [CPU_STEAL] = ColorPair(Cyan, Black),
+      [CPU_GUEST] = ColorPair(Cyan, Black),
+      [PRESSURE_STALL_THREEHUNDRED] = ColorPair(Cyan, Black),
+      [PRESSURE_STALL_SIXTY] = A_BOLD | ColorPair(Cyan, Black),
+      [PRESSURE_STALL_TEN] = A_BOLD | ColorPair(White, Black),
+      [ZFS_MFU] = ColorPair(Blue, Black),
+      [ZFS_MRU] = ColorPair(Yellow, Black),
+      [ZFS_ANON] = ColorPair(Magenta, Black),
+      [ZFS_HEADER] = ColorPair(Cyan, Black),
+      [ZFS_OTHER] = ColorPair(Magenta, Black),
+      [ZFS_COMPRESSED] = ColorPair(Blue, Black),
+      [ZFS_RATIO] = ColorPair(Magenta, Black),
+      [ZRAM] = ColorPair(Yellow, Black),
+   }
 };
 
 int CRT_cursorX = 0;
diff --git a/CRT.h b/CRT.h
index 4806994..9342900 100644
--- a/CRT.h
+++ b/CRT.h
@@ -35,6 +35,7 @@ typedef enum ColorScheme_ {
    COLORSCHEME_MIDNIGHT,
    COLORSCHEME_BLACKNIGHT,
    COLORSCHEME_BROKENGRAY,
+   COLORSCHEME_KEES,
    LAST_COLORSCHEME
 } ColorScheme;
 
diff --git a/ColorsPanel.c b/ColorsPanel.c
index c076adc..1e8eded 100644
--- a/ColorsPanel.c
+++ b/ColorsPanel.c
@@ -37,6 +37,7 @@ static const char* const ColorSchemeNames[] = {
    "MC",
    "Black Night",
    "Broken Gray",
+   "Kees' Pink Twist",
    NULL
 };
 
